processor/stream/package_tests/transaction_attrs_test.go

```diff
@@ -54,13 +54,17 @@ func transactionFieldsNotInPayloadAttrs() *tests.Set {
 		"context.http",
 		"context.http.status_code",
 		"host.ip",
+		"transaction.duration.count",
 		"transaction.marks.*.*",
 		tests.Group("observer"),
 		tests.Group("user"),
 		tests.Group("client"),
 		tests.Group("url"),
 		tests.Group("http"),
 		tests.Group("span"),
+		tests.Group("transaction.self_time"),
+		tests.Group("transaction.breakdown"),
+		tests.Group("transaction.duration.sum"),
```

Comment:
You shouldn't need to exclude `transaction.*` attributes that we expect to be indexed. If that's necessary then this uncovers a real problem. Same for spans above.

Reply:
as per offline discussion: keeping these as they are not in the corresponding `testdata` event